### DISCUSSION TITLE
Is  support  very large dir  and  bucket

### DISCUSSION THREAD

**Discussion by lygn128** [seaweedfs/seaweedfs#4450] [General]
i read the wiki, but have some probleams also, can you  explain to me, thank you very much?
1 after write some data to a bucket, then  i  can access them from fuse? 
2 can i upload many many file( 50PB) to only one bucket?  one bucket's data will storage to many servers?
3  can i upload many many file( 50PB) to only one dir?  one dir's data will storage to many servers? 

thank you very much.

**Comment by LuuucasChen** [comment]
Hi，how do u deal with it? I also need to store a large size of files in one dir by filer, and use oracle to store metadata. Can seaweed support that. Thanks so much!
